frontend/src/pages/admin/OverviewPage/components/CachingPanel.tsx

```diff
@@ -0,0 +1,139 @@
+/* eslint-disable no-return-assign, consistent-return */
+import { useEffect, useRef, useState } from "react";
+import { faRotate } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+import { createNotification } from "@app/components/notifications";
+import { Badge, Button, DeleteActionModal } from "@app/components/v2";
+import { useOrgPermission } from "@app/context";
+import { usePopUp } from "@app/hooks";
+import { useInvalidateCache } from "@app/hooks/api";
+import { useGetInvalidatingCacheStatus } from "@app/hooks/api/admin/queries";
+import { CacheType } from "@app/hooks/api/admin/types";
+
+export const CachingPanel = () => {
+  const { mutateAsync: invalidateCache } = useInvalidateCache();
+  const { data: isInvalidating, refetch: refetchInvalidatingStatus } =
+    useGetInvalidatingCacheStatus();
```

Comment:
There is a refetchInterval option in useQuery

https://tanstack.com/query/v4/docs/framework/react/examples/auto-refetching